frontend/src/components/menu/MenuDrawer.vue

```diff
@@ -154,31 +152,31 @@
           <q-slider
             :value="heatmap_radius"
             :min="1"
-            :max="100"
+            :max="200"
             @input="val => setHeatmapRadius(val)"
           />
         </div>
 
         <q-separator />
 
-        <q-item-label header>
-          Filters
-        </q-item-label>
+        <!--        &lt;!&ndash; TODO remove altogether? &ndash;&gt;-->
+        <!--        <q-item-label header>-->
+        <!--          Filters-->
+        <!--        </q-item-label>-->
 
-        <!-- TODO v-if -->
-        <AirbnbFilters />
+        <!--        <AirbnbFilters v-if="map_type === 'AIRBNB'" />-->
       </q-list>
     </q-scroll-area>
   </q-drawer>
 </template>
 
 <script>
-import AirbnbFilters from "./AirbnbFilters.vue";
+// import AirbnbFilters from "./AirbnbFilters.vue";
 
 export default {
   name: "MenuDrawer",
   components: {
-    AirbnbFilters,
+    // AirbnbFilters,
```

Comment:
what are these filters?

Reply:
Various filters for which Airbnbs to show (e.g. by price, etc.)
I think we can leave them out and add them when we have time to spare, since I don't think this filtering is implemented on the backend side.